### DISCUSSION TITLE
ModuleNotFoundError: No module named 'customtkinter'

### DISCUSSION THREAD

**Discussion by adambelbrik** [TomSchimansky/CustomTkinter#636] [General]
i need help, just installing the library using pip but am getting this error:
ModuleNotFoundError: No module named 'customtkinter'

**Comment by Akascape** [comment]
@adambelbrik 
Try this command:
`py -3.10 -m pip install customtkinter`
where `-3.10` is your python version.

**Reply by adambelbrik** [reply]
@Akascape 
Requirement already satisfied: customtkinter in c:\users\adam\appdata\roaming\python\python311\site-packages (4.6.3)
my python version is 3.11

**Reply by Akascape** [reply]
@adambelbrik I guess you are using a different python version to run the program where customtkinter is not installed.
Install the module in that version also and try.

Or you can also change the environment path.

**Reply by HamzaHamani** [reply]
thank you so much it worked

**Comment by adambelbrik** [comment]
Nevermind, i've added customtkinter folder to my project folder and its working!
Thank you.

**Reply by McTwinky** [reply]
Hi, I am having the same issue, how did you add the customtkinter folder to your proect file?  Where did you find teh customtkinter folder?

**Reply by Akascape** [reply]
@McTwinky There is a 'download zip' option in the front page:
![demo](https://user-images.githubusercontent.com/89206401/204619163-3d25aedb-193b-4b54-9158-72af95fbf7bc.png)

After downloading, extract the file and copy the `customtkinter` folder to the directory where you will work on your project.

**Reply by McTwinky** [reply]
thanks I'll try this

**Reply by McTwinky** [reply]
Hummm, I downloaded the zip file and extracted teh file to my folder and ran teh import customtkinter cmd but I continue to get this error: ModuleNotFoundError: No module named 'customtkinter'

**Reply by McTwinky** [reply]
Hello, thanks for your help, solved.  Found a way to get Tom's customtkinter code into the library. Your help is much appreciated

**Reply by The85Bastel** [reply]
Hi McTwinky I have the same problem. How did you resolved it. I`m running pyhton 3.11. When I use the pip way it never finds the customtkinter module

**Reply by larryhardin** [reply]
> Hello, thanks for your help, solved. Found a way to get Tom's customtkinter code into the library. Your help is much appreciated

And no explanation. This is why communities don't need leeches.

**Comment by McTwinky** [comment]
You helped big time, thanks a million...

**Comment by McTwinky** [comment]
Hi sorry it took me a while to respond. I opened the cmd prompt and
entered: C:\Users\mip XXXXXXXXXXXX -m pip install --upgrade pip.  find your
path to your python install and run the upgrade pip cmd. after it
uninstalled the old pip it auto installed the upgraded pip version. Then I
ran teh cmd in the terminal window:  pip install customtkinter and a moment
later it installed the module: Successfully installed customtkinter-5.0.3
darkdetect-0.8.0.  hope it helps Tom

**Comment by McTwinky** [comment]
sorry Tom, then install customtkinter from the package tab at the bottom of
your IDE

**Comment by McTwinky** [comment]
Hello again, just to let you know that I had to add a customtkinter master
folder to my projects file to get it to work properly.  Let me know if you
want a compressed copy of this file

regards
McTwinky

**Comment by VictoRGome** [comment]
I have the same problem...

**Reply by Akascape** [reply]
@VictoRGome search in this thread: https://github.com/TomSchimansky/CustomTkinter/issues/1915

**Comment by Eugenepython** [comment]
i have same problem

**Comment by imiladi** [comment]
I had the same problem, try changing the python interpreter under view/command palette in vs code

**Comment by sriramgroot** [comment]
I'm too facing this issue. As when I try to install 'customtkinter' module and while running Python file, it throws an error as
```
ModuleNotFoundError: No module named 'customtkinter'
```
